### ISSUE TITLE
jx environment-.._dev repo PR created by boot doesn't update jx-requirements.yml

### ISSUE THREAD

**Issue by cagiti** [jenkins-x/jx#5095]
### Summary 
The `environment-<cluster_name>-dev` repository created when installing a GitOps based cluster doesn't contain all of the boot configuration within the PR.

### Steps to reproduce the behavior
- Create a cluster
- Install Jenkins X using `jx boot`
- Modify the locally downloaded `jenkins-x-boot-config` `jx-requirements.yml` file and add `gitops: true` to the root of the yaml document.
- Re-run `jx-boot`
- The `environment-<cluster_name>-dev` is created within the Organisation name which you selected, however, the PR made against that repository doesn't contain the changes made to the `jx-requirements.yml` file stored locally within the `jenkins-x-boot-config` repository which was download during installation.

### Expected behavior
The local copy of `jx-reqiuirements.yml` is included in the `environment-<cluster_name>-dev` repository PR created when the an installation of a GitOps cluster is completed.

### Actual behavior

The PR contains the following https://github.com/cagiti-org/environment-coops-boot-2-dev/pull/1

### Jx version

The output of `jx version` is:

```
🙅 developer ~/jenkins-x-boot-config(master)$ jx version
NAME               VERSION
jx                 2.0.583
Kubernetes cluster v1.12.8-gke.10
kubectl            v1.15.2
helm client        Client: v2.14.3+g0e7f3b6
git                2.22.0
Operating System   Alpine Linux 3.10.1
WARNING: 
A new jx version is available: 2.0.598
```

### Jenkins type

- [x] Serverless Jenkins X Pipelines (Tekton + Prow)
- [ ] Classic Jenkins

**Comment by pmuir** [comment]
This was happening due to a funky git setup that Cai had. Cai, can you include how to reproduce the git repo setup in the instructions?

**Comment by macox** [comment]
I've just reproduced this by...
- `jx boot` (without the config cloned - so that auto cloning occurs)
- edit config to set `gitops: true`
- rerun `jx boot`

you end up with the 48 file changes (as in https://github.com/cagiti-org/environment-coops-boot-2-dev/pull/1) because the PR is putting everything that should be in the env folder in to the root folder. And locally you have the following files untracked
```     env/.gitignore
        env/OWNERS
        env/README.md
        env/clearcluster.sh
        env/env/
        env/jenkins-x.yml
        env/jx-requirements.yml
        env/kubeProviders/
        env/prowConfig/
        env/repositories/
        env/systems/```

**Comment by macox** [comment]
Fixed by #31 and a duplicate of #4931 I think
